patchwork-god-classes/src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -19,9 +19,27 @@
 
 package net.minecraftforge.event;
 
+import javax.annotation.Nullable;
+
+import net.minecraftforge.common.capabilities.CapabilityDispatcher;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+
+import net.patchworkmc.impl.capability.CapabilityEvents;
+
 /*
  * Note: this class is intended for mod use only, to dispatch to the implementations kept in their own modules.
  * Do not keep implementation details here, methods should be thin wrappers around methods in other modules.
  */
 public class ForgeEventFactory {
+	// TODO: these keeps the forge restriction of T extends ICapabilityProvider, which patchwork otherwise does not enforce
+	// should it be removed?
```

Comment:
yes remove it because it will cause issues for fabric mods trying to call this

Reply:
got it.